src/main/java/org/jboss/snowdrop/licenses/properties/GeneratorProperties.java

```diff
@@ -66,4 +66,9 @@ public GeneratorProperties(String filePath) {
                 .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
     }
 
+    public Optional<String> getLicenseServerUrl() {
```

Comment:
Should either be getLicenseServiceUrl or property changed to LICENSE_SERVER_URL.

Reply:
do you have any preferences for this name?

Reply:
I think getLicenseServiceUrl based on the proeprty name would be fine.

Reply:
ok